com.github.cjbassi.ytop.yaml

```diff
@@ -0,0 +1,35 @@
+app-id: com.github.cjbassi.ytop
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+command: ytop
+finish-args:
+  - --share=network
+build-options:
+  append-path: /usr/lib/sdk/rust-stable/bin
+  env:
+    RUSTFLAGS: --remap-path-prefix =../
+    CARGO_HOME: /run/build/ytop/cargo
+modules:
+  - name: ytop
+    buildsystem: simple
+    build-commands:
+      - cargo --offline fetch --manifest-path Cargo.toml --verbose
+      - cargo --offline build --release --verbose
+      - install -D -m 755 ./target/release/ytop -t /app/bin/
+      - install -D -m 644 ./com.github.cjbassi.ytop.appdata.xml -t /app/share/metainfo/
+      - install -D -m 644 ./com.github.cjbassi.ytop.svg -t /app/share/icons/hicolor/scalable/apps/
+      - install -D -m 644 ./com.github.cjbassi.ytop.desktop -t /app/share/applications/
+    sources:
+      - type: git
+        commit: d4eabcd415dbad33b24e31d3e89b0c79a893a355
+        url: https://github.com/cjbassi/ytop/
+      - type: file
+        path: data/com.github.cjbassi.ytop.appdata.xml
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

Reply:
Only for add-ons https://docs.fedoraproject.org/en-US/packaging-guidelines/AppData/
And appdata files installed in proper `/app/share/metainfo/`.

Reply:
I would follow appstream docs.

Reply:
OK, NP. Will fix this shortly.